plugins/pyenv/pyenv.plugin.zsh

```diff
@@ -22,13 +23,23 @@ fi
 if [[ $FOUND_PYENV -ne 1 ]]; then
     if (( $+commands[brew] )) && dir=$(brew --prefix pyenv 2>/dev/null); then
         if [[ -d $dir/bin ]]; then
-            export PATH="$PATH:$dir/bin"
             FOUND_PYENV=1
+            DISCOVERED_PYENV=1
         fi
     fi
 fi
 
 if [[ $FOUND_PYENV -eq 1 ]]; then
+    if [[ $DISCOVERED_PYENV -eq 1 ]]; then
+        echo pyenv is badly configured.  Add the following lines to your .profile and source it in your .zprofile:
```

Comment:
nitpick: pyenv is not the one being "badly" configured - it's the user's shell.

Reply:
Updated, thanks.